### PR TITLE
Add logging for on_message.

### PR DESCRIPTION
**Pull request by SunDwarf**
Pretending we're a company 101

### CODE REVIEW

**Comment by EJH2** [comment]
Looks good to me.

**Comment by sponges** [comment]
DiscordBans Llc Inc Limited

File: bot.py [hunk 1]
```diff
@@ -58,6 +58,21 @@ def __del__(self):
     async def on_ready(self):
         self.logger.info("Loaded DBansBot, logged in as {0.user.name}#{0.user.discriminator}.".format(self))
 
+    async def on_message(self, message):
+        """
+        Process commands and log.
+        """
+        self.logger.info("Recieved message: {message.content} from {message.author.display_name}{bot}"
```
Comment (gdude2002) [inline bot.py]: I would rather something like..

`{logger.name} | <{user#discrim}{ [BOT]}{ / channel}> {message}`

..Where logger.name is the same as the server name

Comment (SunDwarf) [inline bot.py]: TOO LATE NOW

**Pull request merged**